### ISSUE TITLE
The useful information on the Multisong Leaderboard

### ISSUE THREAD

**Issue by hunterhogan** [ZFTurbo/Music-Source-Separation-Training#87]
### Mail delivery failed: returning message to sender

```
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a permanent error. The following address(es) failed:

  [amvnews@mail.ru](mailto:amvnews@mail.ru)
    (ultimately generated from [turbo@mvsep.com](mailto:turbo@mvsep.com))
    host [mxs.mail.ru](http://mxs.mail.ru/) [94.100.180.31]
    SMTP error from remote mail server after end of data:
    550 Message rejected. Protonmail is blocked

----------------------------------------------
message/delivery-status
----------------------------------------------
Reporting-MTA: dns; [mvsep.com](http://mvsep.com/)

Action: failed
Final-Recipient: [rfc822;turbo@mvsep.com](mailto:rfc822;turbo@mvsep.com)
Status: 5.0.0
Remote-MTA: dns; [mxs.mail.ru](http://mxs.mail.ru/)
Diagnostic-Code: smtp; 550 Message rejected. Protonmail is blocked

----------------------------------------------
text/rfc822-headers
----------------------------------------------
Return-path: <[HunterHogan@pm.me](mailto:HunterHogan@pm.me)>
Received: from [mail-4316.protonmail.ch](http://mail-4316.protonmail.ch/) ([185.70.43.16])
	by [mvsep.com](http://mvsep.com/) with esmtps  (TLS1.3) tls TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
	(Exim 4.95)
	(envelope-from <[HunterHogan@pm.me](mailto:HunterHogan@pm.me)>)
	id 1t5Yyv-002xcy-6x
	for [turbo@mvsep.com](mailto:turbo@mvsep.com);
	Mon, 28 Oct 2024 23:17:05 +0000
```

Dear Bas Curtiz and MVSep,

### [Multisong leaderboard](https://mvsep.com/quality_checker/multisong_leaderboard?sort=vocals&ensemble=0)

Rating | ID | Team and Algo name | Info | Ensemble | SDR Bass | SDR Drums | SDR Other | SDR Vocals | SDR Instrumental
-- | -- | -- | -- | -- | -- | -- | -- | -- | --
1 | 360 | Multisong dataset \| Original  stems \| bass/drums/other joined | ? | - | 73.4643 | 72.8625 | 70.7884 | 71.7201 | 69.3067
2 | 4902 | Multisong dataset \| Original stems \| bass/drums/other joined - MP3 320kbps CBR | ? | - | 62.6150 | 34.9642 | 42.0354 | 37.7338 | 37.8225
3 | 4901 | Multisong dataset \| Original stems \| bass/drums/other joined - MP3 128kbps CBR | ? | - | 25.7775 | 19.7684 | 21.4349 | 20.0551 | 21.3164

I propose that you move this useful information from the Multisong Leaderboard to the [Quality Checker page](https://mvsep.com/quality_checker) or to another place appropriate for reference information.

Some thoughts to consider:

- The information is on a leaderboard of algorithm performance, but the information is not about algorithm performance.
- It can be interpreted to mean that one, two, or three algorithms are very ahead of every other algorithm.
- Alternatively, it can be interpreted to mean that one or more people have an unfair advantage.
- Furthermore, it can be interpreted to mean that one or more people with an unfair advantage are willing to use their unfair advantage to guarantee they possess the highest ranks on the leaderboard.
- Additionally, it can be interpreted to mean that the administrators of the leaderboard will allow people to abuse the system just so they can have the highest ranks on the leaderboard.
- The information is extremely useful, but because it is presented as algorithm entries on the leaderboard, it is difficult to interpret the information and to learn from the information.

**Comment by bascurtiz** [comment]
You are aware you can hover over the question-mark icon to see more details, what this entry is right?
![image](https://github.com/user-attachments/assets/477839d3-27a1-4554-ad64-e8965af9afa5)

**Comment by hunterhogan** [comment]
Yes

(peace) سلام, 和谐 (group harmony), health, and happiness,

Hunter

Sent from Proton Mail Android

-------- Original Message --------

**Comment by bascurtiz** [comment]
Yeah, maybe @ZFTurbo can put them in a static table outside the lists itself on the https://mvsep.com/en/quality_checker page ?
